File: support_tools/swigwin-3.0.12/Examples/test-suite/go/ret_by_value_runme.go
package main

import "./ret_by_value"

func main() {
	a := ret_by_value.Get_test()
	if a.GetMyInt() != 100 {
		panic(0)
	}

	if a.GetMyShort() != 200 {
		panic(0)
	}
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/octave/ret_by_value_runme.m
# do not dump Octave core
if exist("crash_dumps_octave_core", "builtin")
  crash_dumps_octave_core(0);
endif

ret_by_value

a = ret_by_value.get_test();
if (a.myInt != 100)
    error
endif

if (a.myShort != 200)
    error
endif
